service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DirectionAssignee.java

```diff
@@ -3,7 +3,7 @@
 public enum DirectionAssignee {
     ALL_PARTIES("allParties"),
     LOCAL_AUTHORITY("localAuthorityDirections"),
-    PARENTS_AND_RESPONDENTS("parentsAndRespondentsDirections"),
+    PARENTS_AND_RESPONDENTS("respondentDirections"),
```

Comment:
Shall we rename this enum to `RESPONDENTS`?

Reply:
breaking change to cases in AAT and demo, I had it like that, but have reverted. Been open for a few days and no reviews so I thought the BREAKING CHANGE flag scared people off

Reply:
Can confirm was very scared 😨